### ISSUE TITLE
[FEATURE] Make Footer more attractive by adding Social Media Handles

### ISSUE THREAD

**Issue by parthksingh1** [vatsalsinghkv/easy-fix#12]
### Description

Adding Social Media Handles will allow other users to know more about you and about project;
benefits of adding social media handles to a project:

Engagement: Connect with users and foster community discussions around your project.
Visibility: Enhance project visibility as content gets shared or discussed on social platforms.
Feedback: Get direct user feedback and insights to iterate and improve the project.
Drive Traffic: Direct interested users from social platforms to your project page.
Trust: Show that the project is active and updated, building user trust.
Updates: Share project updates, milestones, and news effortlessly.
Networking: Connect with potential collaborators, contributors, or stakeholders.
Linking social media handles in your project helps in bridging the communication gap, expanding reach, and fostering an active community around your project.

### ✨ Browser

Google Chrome

### Checklist 🚀

- [X] I checked and didn't find similar [issue](https://github.com/vatsalsinghkv/easy-fix/issues)
- [X] I have read the [Contributing Guidelines](https://github.com/vatsalsinghkv/easy-fix/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (blank for no).

**Comment by parthksingh1** [comment]
Please Assign me this Issue, will make website look more attractive

**Comment by Clou-D-Walker** [comment]
I will add more attractive footer, add social media handles, ensure responsiveness.

**Comment by 29deepanshutyagi** [comment]
Kindly assign me this issue @vatsalsinghkv   under Hacktoberfest 2023,  i  can customize the code to add more social media platforms, or to change the size, color, and spacing of the icon and  also adds  additional CSS styling to the social media icons section, such as adding a border or a shadow.

**Comment by vatsalsinghkv** [comment]
@parthksingh1 how's the progress

**Comment by Rehan52004** [comment]
Is this issue open.

**Comment by vatsalsinghkv** [comment]
@Rehan52004 @29deepanshutyagi Would any of you guys wanna work on this issue?

**Comment by devesh-mishra13** [comment]
is this issue open i am interested to add the social media icons of the website admin in the footer

**Comment by vatsalsinghkv** [comment]
@devesh-mishra13 are you still interested?

**Comment by devesh-mishra13** [comment]
> @devesh-mishra13 are you still interested?

Yes, @vatsalsinghkv i am interested in making the footer more attractive can you please assign this issue to me.

**Comment by Anurag-singh-thakur** [comment]
hey assign it to me please

**Comment by mohamedaminehamdi** [comment]
I am intrested in this issue assign to me

**Comment by JustAdi10** [comment]
Hey, I'm interested and can get this done within 24 hours could you assign this to me.

**Comment by kartik-212004** [comment]
I'm interested in resolving this issue and making the footer more attractive.

**Comment by sandy4242** [comment]
@vatsalsinghkv hi this issue could also be a great good first issue to work on, and i would love to work on this issue ...
thank you

**Comment by AadityaBansal01** [comment]
can u please assign this to me?

**Comment by sai-maker-cloud** [comment]
Please assign to me
